### DISCUSSION TITLE
Replicate previous +client.ts behavior

### DISCUSSION THREAD

**Discussion by jonaskuske** [vikejs/vike#1686] [Discussion] [answered]
Previously, Vike only ran `+client.ts` scripts in HTML-only mode, when `meta: { Page: { env: { client: false } } }`. This was quite handy for me, as my app can run in different modes and I only want to run some scripts if other client-side code is not available. What's the best way to replicate the previous behavior, where `+client.ts` scripts ran only when other client code / hydration was disabled?

**Comment by brillout** [comment, marked as answer]
Does one of the following work for you?
1. Group them by using https://vike.dev/filesystem-routing#domain-driven-file-structure.
2. Create a `+config.js` file for each of them and set the `client` setting in there.
3. Create a custom boolean setting that automatically applies `2.` (and potentially other things such as `meta.Page.env`).
